tools/monitoring/main.go

```diff
@@ -106,7 +107,19 @@ func testCloudSQLConn(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprintf(w, "Failed to ping the database %v", err)
 		return
 	}
-	fmt.Fprintf(w, "Success\n")
+	fmt.Fprintf(w, "DB Connection Succeeds\n")
+
+	selectedConfig := config.SelectedConfig{}
+
+	db, _ := dbConfig.Connect()
```

Comment:
any reason we are ignoring the err?

Reply:
Because that is already checked by TestConn() before this

Reply:
I think TestConn can be removed from the `testCloudSQLConn` handler and check for `dbConfig.Connect()` error instead.

Reply:
Agree